scripts/spack/packages/adiak/package.py

```diff
@@ -3,7 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
```

Comment:
This is a reversion of a recent change in Spack.  This should be fixed on the LiDO side not ours or better understood why this is a problem.

https://github.com/spack/spack/pull/30404

Reply:
It is being fixed on the LiDO side [here](https://lc.llnl.gov/gitlab/topopt/lido-2.0/-/merge_requests/420). And I thought it'd make sense to apply the fix here, so we don't have to every time LiDO needs a Smith/ Serac update.

I'm not sure what the issue was, but on building the tpls in Serac, it seemed to have made no difference.